### PR TITLE
Migrate data from category to categories

### PR DESCRIPTION
**Pull request by PritishC**

### CODE REVIEW

File: cms.py [hunk 1]
```diff
@@ -1062,3 +1063,29 @@ class ArticleCategoryRelation(ModelSQL):
...
+    def __register__(cls, module_name):
+        TableHandler = backend.get('TableHandler')
+        cursor = Transaction().cursor
+        Article = Pool().get('nereid.cms.article')
+
+        table = TableHandler(cursor, Article, module_name)
+
+        # 3.2 makes a many2many relationship b/w article and categories.
+        # Remove not null constraint from category.
+        if table.column_exist('category'):
```
Comment (prakashpp) [inline cms.py]: remove this

File: cms.py [hunk 2]
```diff
@@ -1062,3 +1063,29 @@ class ArticleCategoryRelation(ModelSQL):
...
+        # Move data from category to categories
+        if table.column_exist('category'):
+            article = Table('nereid.cms.article')
+            article_categ_rel = Table('nereid.cms.category-article')
+
+            article_select = article.select(article.id, article.category)
+            cursor.execute(*article_categ_rel.insert(
+                columns=[article_categ_rel.article, article_categ_rel.category],
+                values=article_select
+            ))
```
Comment (prakashpp) [inline cms.py]: drop the column after the migration

Comment (PritishC) [inline cms.py]: Done.

**Comment by prakashpp** [comment]
LGTM

**Pull request merged**